### PR TITLE
[Doc]: matched 3d plots titles with others

### LINKED ISSUE #26486: [Doc]: match 3D plot types listings titles to other titles
### Documentation Link

https://matplotlib.org/devdocs/plot_types/index.html

### Problem

The plot listings in plots types all have signature + data args except the 3D section. 

### Suggested improvement

The titles of the 3d plots should be updated to match the format of the other titles, which is `function_name(data specific args)`

### PR DESCRIPTION
**Pull request by AmirAflak**
## PR summary
closes #26486
## PR checklist

- [x] "closes #0000" is in the body of the PR description to [link the related issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] new and changed code is [tested](https://matplotlib.org/devdocs/devel/testing.html)
- [x] *Plotting related* features are demonstrated in an [example](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-examples-and-tutorials)
- [x] *New Features* and *API Changes* are noted with a [directive and release note](https://matplotlib.org/devdocs/devel/coding_guide.html#new-features-and-api-changes)
- [x] Documentation complies with [general](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-rest-pages) and [docstring](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-docstrings) guidelines

### CODE REVIEW

File: galleries/plot_types/3D/scatter3d_simple.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """
 ==============
-3D scatterplot
+scatter(x, y, z)
 ==============
```
Comment (QuLogic) [inline galleries/plot_types/3D/scatter3d_simple.py:4]: The `=` above and below need to be the same length as the heading text, in all cases.

**Comment by AmirAflak** [comment]
could you please review my pull request? I have addressed all the requested changes and would appreciate your feedback. Thank you!

**Comment by ksunden** [comment]
I think this may have been closed automatically because you opened another PR from `main`, we recommend using a feature branch so that this does not happen.

**Comment by QuLogic** [comment]
Replaced by #26571.

**Pull request closed without merging**